nvflare/app_opt/tracking/wandb/wandb_writer_metrics_exchanger.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Optional
+
+from nvflare.apis.analytix import AnalyticsDataType
+from nvflare.app_common.tracking.log_writer import LogWriter
+from nvflare.app_common.tracking.tracker_types import LogWriterName
+
+
+class WandBWriterForMetricsExchanger(LogWriter):
```

Comment:
For these sub-Writers, suggest merge them as well.

Underlying they should all call self.log

And they can all remove __init__ method because it is the same as the parent's init method.